cruditor/forms.py

```diff
@@ -20,6 +20,13 @@ class CruditorFormsetMixin(object):
     """
     js_formset_options = None
 
+    @cached_property
```

Comment:
cached_property is only available for Python 3.8+. We should leverage the `django.utils.functional.cached_property` instead.